Src/Support/Google.Apis.Auth.Tests/OAuth2/GoogleCredentialTests.cs

```diff
@@ -476,5 +476,40 @@ private void AssertContainsQuotaProject(IEnumerable<KeyValuePair<string, IEnumer
             string quotaProject = Assert.Single(headerValues.Value);
             Assert.Equal(expectedValue, quotaProject);
         }
+
+        [Fact]
+        public void TestImpersonatedCredential()
```

Comment:
Nit: The "Test" prefix here is unconventional. I'd just call this `Impersonate` (as that's the method we're testing) then `CreateScoped_Impersonated` and `CreateWithQuotaProject_Impersonated` as the tests below.

Reply:
done